src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleEqualityTests.cs

```diff
@@ -1659,15 +1662,24 @@ static void Main()
 }";
             var comp = CreateCompilation(source);
             comp.VerifyDiagnostics(
-                // (6,30): error CS0019: Operator '==' cannot be applied to operands of type '<null>' and 'lambda expression'
+                // (6,65): error CS8652: The feature 'inferred delegate type' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
```

Comment:
Could you also verify diagnostics with LangVer=preview? (also consider doing the same in `TestTypelessTupleAndTupleOfDefaults` above, and `TestWithTypelessTuple` below)